### ISSUE TITLE
Upload to PyPI

### ISSUE THREAD

**Issue by sephalon** [pengutronix/usbsdmux#19]
To simplify the installation, it would be nice to have an `usbsdmux` package on the Python Package Index.

#17 must be merged first. Metadata in `setup.py` should be completed according to the [Python Packaging User Guide](https://packaging.python.org/guides/distributing-packages-using-setuptools/#setup-args). The release should also be VCS-tagged accordingly.

**Comment by SmithChart** [comment]
usbsdmux can now be installed from PyPI: https://pypi.org/project/usbsdmux/

I will continue to update the documentation where needed.
